src/test/java/seedu/exercise/logic/commands/SuggestPossibleCommandTest.java

```diff
@@ -17,15 +29,22 @@
 import seedu.exercise.model.ModelManager;
 import seedu.exercise.model.ReadOnlyResourceBook;
 import seedu.exercise.model.UserPrefs;
+import seedu.exercise.model.property.CustomProperty;
 import seedu.exercise.model.property.Muscle;
 import seedu.exercise.model.resource.Exercise;
+import seedu.exercise.testutil.CustomPropertyBuilder;
 
 public class SuggestPossibleCommandTest {
 
     private Model model;
     private Model expectedModel;
     private Set<Muscle> targetMuscles;
     private Map<String, String> targetCustomProperties;
+    private Predicate<Exercise> predicateMuscleAnd;
+    private Predicate<Exercise> predicateMuscleOr;
+    private Predicate<Exercise> predicateCustomPropertyAnd;
+    private Predicate<Exercise> predicateCustomPropertyOr;
+    private boolean isStrict = true;
```

Comment:
can these be local variables instead? does not seem like they need to be class variables for tests

Reply:
ok